### DISCUSSION TITLE
posts/n-2n%E9%97%B4%E5%AD%98%E5%9C%A8%E7%B4%A0%E6%95%B0

### DISCUSSION THREAD

**Discussion by giscus** [Locietta/blog-lost-pieces#7] [Announcements]
# posts/n-2n%E9%97%B4%E5%AD%98%E5%9C%A8%E7%B4%A0%E6%95%B0

初中的时候瞎琢磨过的问题，现在捡起来看还是很有趣

https://blog.locietta.xyz/posts/n-2n%E9%97%B4%E5%AD%98%E5%9C%A8%E7%B4%A0%E6%95%B0

**Comment by Dchenc12** [comment]
牛哇
